tests/unit_tests/test_wave_06.py

```diff
@@ -228,16 +219,17 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert not result
```

Comment:
Instead of checking that `result` is a falsey value, we should check that the result is an empty list because the README says the function returns an empty list if no items match the category.